core/lib/DomEnv.ts

```diff
@@ -154,10 +154,10 @@ const typesonRegistryScript = fs.readFileSync(
 );
 
 const domStorageScript = fs.readFileSync(
-  require.resolve(`@secret-agent/injected-scripts/dist/scripts/domStorage.js`),
+  require.resolve(`@secret-agent/injected-scripts/scripts/domStorage.js`),
```

Comment:
Does this show errors in WebStorm since it doesn't know about js build files from within Typescript files?

Reply:
It doesn't right now